scripts/ide-run-scss-lint-from-docker.sh

```diff
@@ -1,9 +1,3 @@
 #!/usr/bin/env bash
-
-# This script will run scss-lint
-
-PROJECT_DIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )/.." && pwd)
-
-docker run --rm \
-  -v $PROJECT_DIR:/style-guide \
-  brainly/style-guide bash -c "scss-lint /style-guide/$1 -f JSON | sed 's/\/style-guide\///g'"
+SCRIPT_DIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
+source ${SCRIPT_DIR}/run-scss-lint.sh $1 -f JSON
```

Comment:
I can't understand why you split it, also naming doesn't make it easier to get as docker-level instructions are actually in `/run-scss-lint.sh` while this file abstracts that away and doesn't even mention word `docker` at all but is still called `ide-run-scss-lint-from-docker` ? :)